analysis_options.yaml

```diff
@@ -1,229 +1,246 @@
 # Specify analysis options.
 #
-# This file is a copy of analysis_options.yaml from flutter repo
-# as of 2022-07-27, but with some modifications marked with
-# "DIFFERENT FROM FLUTTER/FLUTTER" below. The file is expected to
-# be kept in sync with the master file from the flutter repo.
+# This file is a copy of analysis_options.yaml from flutter repo as of
+# 2022-07-27, but with some modifications marked with "DIFFERENT FROM
+# FLUTTER/FLUTTER" below. The file is expected to be kept in sync with the
+# main file from the flutter repo.
```

Comment:
Will this make it a lot harder to sync over the lint settings from flutter/flutter next time we want to do that? Or does the tooling help with that?